frontend/src/components/atoms/IconButton/IconButton.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+function getRgbValues(color) {
```

Comment:
All colors should be stored in theme object. In our component we will use only reference to this color definition. 
 for example: background-color: ${({ secondary }) => (secondary ? ${theme.colors.primary} : ${theme.colors.secondary})}; 
 
 please remove this method

Reply:
removed